pkg/lightning/config/config.go

```diff
@@ -1107,13 +1112,20 @@ type TikvImporter struct {
 	// default is PausePDSchedulerScopeTable to compatible with previous version(>= 6.1)
 	PausePDSchedulerScope PausePDSchedulerScope `toml:"pause-pd-scheduler-scope" json:"pause-pd-scheduler-scope"`
 	BlockSize             ByteSize              `toml:"block-size" json:"block-size"`
+
+	ChunkSize     int    `toml:"chunk-size" json:"chunk-size"`
+	ChunkCacheDir string `toml:"chunk-cache-dir" json:"chunk-cache-dir"`
 }
 
 func (t *TikvImporter) adjust() error {
 	if t.Backend == "" {
 		return common.ErrInvalidConfig.GenWithStack("tikv-importer.backend must not be empty!")
 	}
+
 	t.Backend = strings.ToLower(t.Backend)
+	if t.Backend == BackendRemote {
+		return common.ErrInvalidConfig.GenWithStack(`tikv-importer.backend = "remote" is not supported`)
```

Comment:
I wonder how your team will use remote backend since there is error 😂

Reply:
I will the check after we complete the remote backend.